db_service/dao.go

```diff
@@ -13,19 +13,19 @@
 // limitations under the License.
 
 // Code generated by go generate; DO NOT EDIT.
-// This file was generated at 2018-08-22 08:41:25.282061797 -0700 PDT m=+0.027525077.
```

Comment:
Removed these lines so running `go generate` will be idempotent which is useful because we have other generated code in the project too.